apps-rendering/src/components/Dateline/index.tsx

```diff
@@ -17,6 +18,7 @@ import { darkModeCss as darkMode } from 'styles';
 interface Props {
 	date: Option<Date>;
 	format: ArticleFormat;
+	edition?: Edition;
```

Comment:
ideally, this would not be optional (as we set a default value in `server/page.tsx`). Setting it as optional here means we don't have to pass `edition` to every instance of this component yet